test-case/manual/wov-script/wov-test.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+rm -rf blob
+preamble_time=0
+history_depth=0
+format=s16_le
+buffer_size=68000
+max_loop=100
+
+usage(){
+        echo "Wrong parameters, please specify the test item"
+        echo "Eg: '-p' preamble_time, '-h' history_depth, '-f' format and '-s' period-size\
+        ./$0 -p 2500 -h 3000 -f s16_le -s 500"
+}
+
+# DEC to HEX
+dec2hex(){
+        printf "%x" $1
+}
+
+_save_def(){
+	echo "Save WoV as default configure blob:"
+	if [ ! -f detector-default-config ]; then
+		sudo sof-ctl -Dhw:0 -n 163 -br -o detector-default-config
```

Comment:
Control ID is not fixed and it's different on each platform.
So please don't use hard code. You can use amixer to get the ID number:
eg. 
$ amixer controls |grep "DETECT10.0 Detector Config"
numid=162,iface=MIXER,name='DETECT10.0 Detector Config'